apps/backend/api/models/Group.js

```diff
@@ -988,6 +1011,39 @@ module.exports = bookshelf.Model.extend(merge({
     }
   },
 
+  // create a calendar subscription for group events
+  async createGroupEventsCalendarSubscriptions ({ groupId, eventChanges }) {
+    const group = await Group.find(groupId)
+    if (!group) return
+
+    if (!group.get('calendar_token')) {
+      await group.save({ calendar_token: uuidv4() }, { patch: true })
+      await group.refresh()
+    }
+
+    // Fetch all events for this group
+    const events = await group.posts().where('posts.type', 'event').fetch()
+
+    // Create the calendar and add the events
+    const cal = ical()
```

Comment:
we need to add a name to the calendar, otherwise it appears in the calendar app using the filename as the calendar name which is not pretty. I added this to the RSVP calendar: 

    const cal = ical({ name: 'My Hylo Events', description: 'All the events I have RSVPed to on Hylo' })

In this case I think it could be     const cal = ical({ name: `All Events for ${this.get('name')}`, description: `All the events in group ${this.get('name')} on Hylo` })